bitbots_blackboard/src/bitbots_blackboard/capsules/world_model_capsule.py

```diff
@@ -243,9 +244,15 @@ def pose_callback(self, pos: PoseWithCovarianceStamped):
         self.pose = pos
 
     def get_current_position(self):
-        orientation = self.pose.pose.pose.orientation
+        try:
+            # get the most recent transform
+            transform = self.tf_buffer.lookup_transform('map', 'base_footprint', rospy.Time())
```

Comment:
For showing that the most recent transform is used maybe use this:
```suggestion
            transform = self.tf_buffer.lookup_transform('map', 'base_footprint', rospy.Time(0))
```